api/src/main/java/me/lokka30/treasury/api/economy/response/FailureReason.java

```diff
@@ -68,31 +72,31 @@
 
     /**
      * A constant representing failure due to a user already being a member of a
-     * {@link me.lokka30.treasury.api.economy.account.BankAccount BankAccount}.
+     * {@link NonPlayerAccount BankAccount}.
      *
      * @since {@link me.lokka30.treasury.api.economy.misc.EconomyAPIVersion#v1_0 v1.0}
      */
     ALREADY_MEMBER_OF_BANK_ACCOUNT,
 
     /**
      * A constant representing failure due to a user already being an owner of a
-     * {@link me.lokka30.treasury.api.economy.account.BankAccount BankAccount}.
+     * {@link NonPlayerAccount BankAccount}.
      *
      * @since {@link me.lokka30.treasury.api.economy.misc.EconomyAPIVersion#v1_0 v1.0}
      */
     ALREADY_OWNER_OF_BANK_ACCOUNT,
```

Comment:
Owners no longer exist in the context of bank accounts. We miss the removal of this failure entry